tests/rt.sh

```diff
@@ -9,18 +9,16 @@ die() { echo "$@" >&2; exit 1; }
 usage() {
   set +x
   echo
-  echo "Usage: $0 -c | -e | -f | -h | -k | -l <file> | -m | -n <name> | -r | -s"
+  echo "Usage: $0 -c | -e | -h | -k | -l <file> | -m | -n <name> | -r "
```

Comment:
we did not get to discuss about the full test feature. We may want to bring "-f" feature back when we do full tests with several rt.conf files if we decided to combine them together, or we can set up a script to run those configurations seperately.

Reply:
It's really a question about complexity in `rt.sh` versus running two steps manually, first `rt.sh` without `-l ...`, then `rt.sh -l rt_35d.conf`. I am leaning towards keeping `rt.sh` (as) simple (as possible), but I would like to hear other opinions.

Reply:
I agree. We should try to keep rt.sh as simple as possible, without adding (or maintaining) unnecessary logic.